web/pages/api/contact.js

```diff
@@ -24,11 +24,13 @@ export default async function handler(req, res) {
         `<p>Name: ${req.body.name}</p>
         <p>Email: ${req.body.emailAddress}</p>
         <p>Phone: ${req.body.phoneNumber}</p>
-        <p>Message: ${req.body.message}</p>`,
+        <p>Message: ${req.body.message}</p>
+        <p>Preference: ${req.body.preference}</p>`,
```

Comment:
@bwreDev We should probably serialize this a little better. Probably just a join with commas would be fine for now